src/JsonSchema/TypeFactory.php

```diff
@@ -102,7 +141,7 @@ private function getClassType(?string $className, string $format = 'json', ?bool
 
         // Skip if $schema is null (filters only support basic types)
         if (null === $schema) {
-            return ['type' => 'string'];
+            return ['type' => 'object'];
```

Comment:
I fear it will break our support for filters (query parameters): we use JSON Schema to describe the expected format of parameters, and in an URL context it must be string by default.

Reply:
I can revert this, since most of the embedded payload items are going to be `$ref` anyway, and `stdClass` can be mapped explicitly (instead of using `object`)

Reply:
I'll revert this, and open a new issue about it.

Reply:
Reverted, deferred to https://github.com/api-platform/core/issues/3403